keyboards/chimera_ortho/readme.md

```diff
@@ -0,0 +1,19 @@
+# Chimera Ortho
+
+![Chimera Ortho](https://imgur.com/pbdNsoP.jpg)
+
+A split wireless 40% ortholinear keyboard 
+
+Keyboard Maintainer: [William Wilson](https://github.com/GlenPickle)  
+
+
+Hardware Supported: Chimera Ortho PCB, WaveShare core nRF51822
+
+Hardware Availability: [Gerbers](https://github.com/GlenPickle/Chimera/ortho/gerbers)
```

Comment:
Looks like this should be https://github.com/GlenPickle/Chimera/tree/master/ortho/gerbers right?

Reply:
yep, fixed. Sorry about that, and happy Halloween!